packages/reporter/src/test/test.tsx

```diff
@@ -18,12 +19,24 @@ interface StudioControlsProps {
   model: TestModel
 }
 
+interface StudioControlsState {
+  copySuccess: boolean
+}
+
 @observer
-class StudioControls extends Component<StudioControlsProps> {
+class StudioControls extends Component<StudioControlsProps, StudioControlsState> {
   static defaultProps = {
     events,
   }
 
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      copySuccess: false,
+    }
+  }
```

Comment:
```suggestion
  state = {
    copySuccess: false,
  }
```

More concise way to create initial state.